driver/config/provider.go

```diff
@@ -88,14 +88,19 @@ const (
 	KeyOAuth2GrantJWTIDOptional                  = "oauth2.grant.jwt.jti_optional"
 	KeyOAuth2GrantJWTIssuedDateOptional          = "oauth2.grant.jwt.iat_optional"
 	KeyOAuth2GrantJWTMaxDuration                 = "oauth2.grant.jwt.max_ttl"
-	KeyRefreshTokenHookURL                       = "oauth2.refresh_token_hook" // #nosec G101
+	KeyRefreshTokenHookURL                       = "oauth2.refresh_token_hook"      // #nosec G101
+	KeyAuthorizationCodeTokenURL                 = "oauth2.authorization_code_hook" // #nosec G101
```

Comment:
```suggestion
	KeyAuthorizationCodeHookURL                 = "oauth2.authorization_code_hook" // #nosec G101
```

Reply:
Not sure if it is a typo, but it should be consistent with other keys that represent hook URLs

Reply:
Was a typo, should be fixed.